util/gpu_usage.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
```

Comment:
Wanted to keep it more separated, util/gpu.py being for the moment more related to tensorflow, but we can move it if you want.